inc/class-cloner.php

```diff
@@ -290,9 +304,6 @@ public function cloneTerm( $term_id ) {
 		// Set endpoint
 		$endpoint = $term['taxonomy'];
 
-		// Get links
-		$links = array_pop( $term );
```

Comment:
We actually need to do this, IIRC. Or remove `_links` [here](https://github.com/pressbooks/pressbooks/pull/902/files#diff-f350d36210bbc147da3ecf1976dbdbc4L297) instead.